layersvt/api_dump.h

```diff
@@ -367,6 +432,102 @@ class ApiDumpSettings {
             return default_value;
     }
 
+    struct NumberToken {
+        uint64_t value;
+        int length;
+    };
+
+    inline NumberToken parseNumber(std::string str, int current_char) {
+        int length = 0;
+        while (current_char + length < str.size() && str[current_char + length] >= '0' && str[current_char + length] <= '9') {
+            length++;
+        }
+        if (length > 0) {
+            uint64_t value = std::atol(&str[current_char]);
+            return NumberToken{value, length};
+        } else {
+            return NumberToken{0, -1};
+        }
+    }
+
+    inline void printErrorMsg(const char *msg) {
+#ifdef ANDROID
+        __android_log_print(ANDROID_LOG_DEBUG, "api_dump", "%s", msg);
+#else
+        fprintf(stderr, "%s", msg);
+#endif
+    }
+
+    inline bool parseConditionalFrameRange(std::string range_str) {
+        int current_char = 0;
```

Comment:
This should probably be `size_t` because some compilers may throw a fit on line 469, 480, etc because int and size_t could be different sizes.

Reply:
Thanks, added.

Reply:
fixed